schemas/com.snowplowanalytics.mobile/diagnostic_error/jsonschema/1-0-0

```diff
@@ -0,0 +1,35 @@
+{
+  "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+  "description": "Schema for a tracker error",
+  "self": {
+    "vendor": "com.snowplowanalytics.mobile",
+    "name": "diagnostic_error",
+    "format": "jsonschema",
+    "version": "1-0-0"
+  },
+  "type": "object",
+  "properties": {
+    "message": {
+      "description": "Message of the error.",
+      "type": "string",
+      "maxLength": 2048
```

Comment:
For all of these, if they can have a minimum Length it's generally preferable to have that too.